### ISSUE TITLE
The requested URL /m was not found on this server

### ISSUE THREAD

**Issue by UnixCro** [CocoaDebug/GoogleTranslate#6]
Hello, I use your app every day. I like her very much. Maybe you could revise it. I don't think it works anymore.

Thank you for your effort

![Bildschirmfoto 2021-04-17 um 11 22 58](https://user-images.githubusercontent.com/70098046/115108228-8cf53b00-9f6f-11eb-98ee-f1af23b7fc6b.jpg)

**Comment by CocoaDebug** [comment]
It is weird, Everything works fine here

![Xnip2021-04-25_14-19-18](https://user-images.githubusercontent.com/6781027/115983015-60759a80-a5d1-11eb-8043-e56dd2c53980.png)

**Comment by CocoaDebug** [comment]
Maybe the google service is unavailable in your area.

 - Source: https://translate.google.com/
 - (Chinese version) Source: https://translate.google.cn/

**Comment by UnixCro** [comment]
Right, https://translate.google.com/ this link works fine, the other one doesn't.  Could you please fix it so I can download and work with your app again.

**Comment by CocoaDebug** [comment]
There are two different download links in [README](https://github.com/CocoaDebug/GoogleTranslate) already. Please check it.

Source: https://translate.google.com/
(Chinese version) Source: https://translate.google.cn/

**Comment by UnixCro** [comment]
Hi, just wanted to say that I just downloaded the correct version of the ReadMe and the program is now working fine. Except that texts cannot be copied. macOS 11.2
![Bildschirmfoto 2021-09-03 um 16 39 42](https://user-images.githubusercontent.com/70098046/132023736-bd3312a6-37ad-43e3-ad9e-03379bd53cd7.png)
